ui/src/app/applications/components/application-details/application-resource-filter.tsx

```diff
@@ -153,16 +153,19 @@ export const Filters = (props: {
             })}
             {namespaces.length > 1 && ResourceFilter({label: 'NAMESPACES', prefix: 'namespace', options: (namespaces || []).filter(l => l && l !== '').map(toOption), field: true})}
             {(tree.orphanedNodes || []).length > 0 && (
-                <div className='filter'>
+                <div className={'filter' + ' ' + `filter__item ${pref.orphanedResources ? 'filter__item--selected' : ''}`}>
```

Comment:
You already have string template literals, so we can put everything inside of it like this instead of part string part template literals:
```<div className={`filter filter__item ${pref.orphanedResources ? 'filter__item--selected' : ''}`}>```

Reply:
Thanks Yi, fixed it!